web/src/layout/controlPanel/repositories/BadgeModal.tsx

```diff
@@ -14,8 +14,9 @@ interface Props {
 }
 
 const BadgeModal = (props: Props) => {
-  const badgeImage = `https://img.shields.io/static/v1?style=flat&label=Artifact%20HUB&labelColor=659dbd&color=39596c&logo=data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNCIgaGVpZ2h0PSIxNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9IiNmZmZmZmYiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIiBjbGFzcz0iZmVhdGhlciBmZWF0aGVyLWhleGFnb24iPjxwYXRoIGQ9Ik0yMSAxNlY4YTIgMiAwIDAgMC0xLTEuNzNsLTctNGEyIDIgMCAwIDAtMiAwbC03IDRBMiAyIDAgMCAwIDMgOHY4YTIgMiAwIDAgMCAxIDEuNzNsNyA0YTIgMiAwIDAgMCAyIDBsNy00QTIgMiAwIDAgMCAyMSAxNnoiPjwvcGF0aD48L3N2Zz4K&logoWidth=18&message=${props.repository.name}`;
-  const badgeLink = `[![Artifact HUB](${badgeImage})](${window.location.origin}/packages/search?repo=${props.repository.name})`;
+  const origin = window.location.origin;
+  const badgeImage = `https://img.shields.io/endpoint?url=${origin}/badge/repository/${props.repository.name}`;
+  const badgeLink = `[![Artifact HUB](${badgeImage})](${origin}/packages/search?repo=${props.repository.name})`;
```

Comment:
Why are you using the link to search results instead of a link to the package?

Reply:
Because it's actually a repository badge, and that link should display all packages available in the repository. In some cases, when there is only a single package in a repository, it may make sense to replace the link with one pointing to the package though.